org.winehq.Wine.yml

```diff
@@ -293,6 +293,21 @@ modules:
       - /lib32/samba
     sources: *samba-sources
 
+  - name: unixodbc
+    sources: &unixodbc_sources
+      - type: archive
+        url: http://www.unixodbc.org/unixODBC-2.3.9.tar.gz
+        sha256: 52833eac3d681c8b0c9a5a65f2ebd745b3a964f208fc748f977e44015a31b207
+    cleanup: &unixodbc_cleanup
+      - /share
```

Comment:
I guess we don't need programs shipped with unixODBC, i.e. `/bin` can be added to `cleanup`?